dot/utils_test.go

```diff
@@ -86,3 +90,105 @@ func TestNewRuntimeFromGenesis(t *testing.T) {
 	_, err := wasmer.NewRuntimeFromGenesis(gen, &wasmer.Config{})
 	require.NoError(t, err)
 }
+
+func TestDeepCopyVsSnapshot(t *testing.T) {
```

Comment:
is it possible to change this into a test benchmark? eg. something like this https://golang.org/pkg/testing/#hdr-Benchmarks

Reply:
We can but for loading genesis file the code is tightly coupled to use `testing.T` struct. We can modify the function to take `require.TestingT` interface as input but it will be major refactor. I can do this in a separate PR.